.github/workflows/coding.yml

```
@@ -50,9 +51,64 @@
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          # files: ./programming/typescript/challenges/test-results/junit/**/*.xml
+      # # Deploy to Github Pages
+      # - name: Deploy to Github Pages
+      #   if: always()
+      #   uses: JamesIves/github-pages-deploy-action@4.1.5
+      #   with:
+      #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      #     BRANCH: gh-pages
+      #     FOLDER: programming/typescript/challenges/test-results
+      #     CLEAN: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)